lib/pbench/server/api/resources/query_apis/datasets_detail.py

```diff
@@ -56,7 +56,7 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
                 data. If "user": None is specified, then only public datasets
                 will be returned.
 
-            "name" is the name of a Pbench agent dataset (tarball).
+            "dataset_name" is the name of a Pbench agent dataset (tarball).
```

Comment:
`s/_name//`

Reply:
Oh, is this what was confusing? That good old `ex` command structure could be easily misinterpreted if you look at it quickly! That's "replace the string `_name` by nothing"; in other words, leave just `dataset`. Ah, well. Sorry for the brevity!